bin/kano-signal

```diff
@@ -25,6 +25,9 @@ import sys
 import os
 import stat
 
+# verbose flag for debugging on the console
+verbose=False
```

Comment:
Is there any way to set this to `True` without changing this file, i.e. through a CLI parameter or system variable?